metaci/testresults/robot_importer.py

```diff
@@ -284,7 +309,7 @@ def render_robot_test_xml(root, test):
     return re.sub(r"sid=.*<", "sid=MASKED<", test_xml)
 
 
-def export_robot_test_results(flowtest) -> None:
+def export_robot_test_results(flowtest, test_results) -> None:
```

Comment:
Is there a reason this is `flowtest` instead of `flowtask`, or is that just a typo?

Reply:
looks like a typo to me.

also please put a docstring to remind us what this does

Reply:
good catch